src/fontra/views/editor/scene-controller.js

```diff
@@ -679,17 +679,17 @@ export class SceneController {
       {
         title: translate("action.break-contour"),
         enabled: () => this.contextMenuState.pointSelection?.length,
-        callback: () => this.breakContour(),
+        callback: () => this.doBreakSelectedContours(),
       },
       {
         title: translate("action.reverse-contour"),
         enabled: () => this.contextMenuState.pointSelection?.length,
-        callback: () => this.reverseSelectedContoursDirection(),
+        callback: () => this.doReverseSelectedContoursDirection(),
```

Comment:
The previous name was already awkward, let's change it to `doReverseSelectedContours`.

Reply:
ok, done.